### ISSUE TITLE
[Quest][Blood Elf Starting Area] Double Quests from Watcher Solanian

### ISSUE THREAD

**Issue by Twiiiiin** [holydeew/WoWHellgarve#158]
Quest/Spell/Item/Creature(ID) The ID is:
https://www.wowhead.com/quest=37443/solanians-belongings
https://www.wowhead.com/quest=37442/the-shrine-of-dathremar
(some others)

The Current Behaviour is as following:
Speak with Watcher Solanian, the quests are doubled, the doubled one is not trackable.
Complete the mission, when you turn it in, the "Complete Mission" button is greyed out and doesn't work
Expected Behaviour should be as following:
You can click the "Complete mission" button and end the mission

**Comment by warscraft360** [comment]
yeah i already add that to my todo list idk how to fix it now but im sure we'll figure out rapidly ^^
